platform/about-these-features/multi-node.md

```diff
@@ -30,32 +30,42 @@ Starting a multi node run in CLI is the same as how you run any other script, Gr
 
 For example --gpus 5 --instance\_type 4\_v100\_32gb will automatically trigger multi node training
 
-Example of 1 node 4 GPUs :
+Prior to being able to run the below you will need to clone a fork of the Pytorch-lightning repo (https://github.com/filintod/pytorch-lightning). For convenience the code to do so is below.
 
-```bash
-grid run --instance_type 4_v100_16gb --framework lightning --gpus 4 torch_elastic_vae.py --data_path grid:stl10:1
+```
+git clone https://github.com/PyTorchLightning/pytorch-lightning.git
+cd pytorch_lightning
+printf "torchvision==0.11.1 \njsonargparse==4.1.0" >> requirements.txt
 ```
 
-Example of 2 nodes 8 GPUs:
+Example of 1 node 1 GPU :
 
 ```bash
-grid run --instance_type 4_v100_16gb --framework lightning --gpus 8 torch_elastic_vae.py --data_path grid:stl10:1
+grid run \
+--name single-node-test \
+--strategy grid_search \
+--instance_type g4dn.xlarge \
+--framework lightning \
+--gpus 1 \
+pl_examples/basic_examples/mnist_examples/image_classifier_5_lightning_datamodule.py \
+--trainer.gpus 1 \
+--trainer.strategy 'ddp' \
+--trainer.max_epochs 50
 ```
```

Comment:
@oojo12 nice! i'm assuming you tested this out? we should add a screenshot of the completed experiment